app/src/main/java/com/example/warriorsocial/ui/login/LoginActivity.java

```diff
@@ -172,6 +177,46 @@ public void onComplete(@NonNull Task<AuthResult> task) {
             }
         });
 
+        forgotPass.setOnClickListener(new View.OnClickListener() {
```

Comment:
I'm guessing this is the part that Richard added for the password reset? 
All looks good to me but some comments could be helpful.